packages/framework-core/src/services/pub-sub/read-model-pub-sub.ts

```diff
@@ -44,8 +44,12 @@ function filterReadModel(readModel: Record<string, any>, filters?: Record<string
     return true
   }
   for (const filteredProp in filters) {
-    const readModelPropValue = readModel[filteredProp]
-    return filterByOperation(filters[filteredProp], readModelPropValue)
+    if (Array.isArray(readModel[filteredProp])) {
+      return readModel[filteredProp].some((readModelFilteredProp: any) =>
+        filterByOperation(filters[filteredProp], readModelFilteredProp)
+      )
+    }
+    return filterByOperation(filters[filteredProp], readModel[filteredProp])
```

Comment:
It would be nice to add a couple of tests for this new scenario...

Reply:
Good point. Allow me to remove this rather unrelated part from the PR due to the lack of time to extend this part right now, at least if the main fix (`graphQLPropType = this.getGraphQLTypeFor(param.type)`) would still be useful to merge.

Reply:
As an alternative, what if you create an issue to implement these tests? Maybe there's someone in my team that can address them...